scripts/docker/Dockerfile.nvcc

```diff
@@ -1,15 +1,18 @@
 ARG BASE=nvidia/cuda:9.2-devel
 FROM $BASE
 
+ARG ADDITIONAL_PACKAGES
+
 RUN apt-get update && apt-get install -y \
         bc \
         wget \
         ccache \
+        $ADDITIONAL_PACKAGES \
         && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-ARG CMAKE_VERSION=3.10.3
+ARG CMAKE_VERSION=3.17.3
```

Comment:
Why would you change the default?

Reply:
I need `3.17` for `CMake` suggesting `-std=c++17` instead of `-std=c++1z` for `gcc-8` when requesting C++17 support. We are using `CMake-3.10.3` for all the other Dcokerfilers so I don't see much of a problem using something different here.

Reply:
So?  It is an argument.  You can specify this version for the build that requires it, you don't need to change for all.